packages/cli-lib/lang/en.lyaml

```diff
@@ -1054,6 +1055,11 @@ en:
             general: "[--general] Tell us about your experience with HubSpot's developer tools"
           bugPrompt: "Create an issue on Github in your browser?"
           generalPrompt: "Open the projects feedback form in your browser?"
+        buildIdPrompt:
+          enterBuildId: "[--build] Deploy which build? (26)"
```

Comment:
What's the purpose of the (26) here? Is it just to show that we're asking for a number? One suggestion is to instead to something like:

`[--build] Enter the number of the build you would like to deploy:`

I'm also wondering if we should support some way for them to easily choose the latest build (without knowing the exact number of it). Maybe that's not worthwhile though.